docs/src/components/TrustTriangleDiagram.css

```
@@ -0,0 +1,94 @@
+  stroke-width: 1;
+}
+
+[data-theme='dark'] .tt-diagram__holder rect,
+[data-theme='dark'] .tt-diagram__verifier rect {
+  fill: rgba(255, 255, 255, 0.04);
+  stroke: rgba(255, 255, 255, 0.12);
+}
+
+/* Edges */
+.tt-diagram__edges line {
+  stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)